content/updates/2020-10/8-New-filters-for-rules-conditions.md

```diff
@@ -8,4 +8,16 @@ pim_announcement_audience:
 ::: meta-data type="New" features="Automation, Productivity" available="early October" link-to-doc="../articles/.html"
 :::
 
-TODO
+In the Rules engine, create more efficient rules conditions thanks to these 4 new filters:
+- On the entity type, i.e., to choose whether to target product models or products,
+- On family variants,
+- On product or product model parents,
```

Comment:
```suggestion
- parents: to target specific product parents or products without parents
```
I'd like to explain the use case for each filter.. but it's not that easy...

Reply:
Yes, that's what I try to do at first, it was tricky, so I stopped.